build.sh

```diff
@@ -35,67 +72,113 @@ do
   esac
 done
 
-if [ ! -d .cldr-data ]; then
-  $QUIET || echo "Checking out CLDR..."
-  svn $($VERBOSE && echo "-v") \
-    checkout http://www.unicode.org/repos/cldr/trunk/common/main/ .cldr-data
+function cloneHtml {
+  HTML_SOURCE=${HTML_SOURCE:-./html}
+  git clone $HTML_GIT_CLONE_OPTIONS \
+    $($VERBOSE && echo "--verbose" || $QUIET && echo "--quiet") \
+    https://github.com/whatwg/html.git $HTML_SOURCE
+}
+if [ -z "$HTML_SOURCE" ]; then
+  $QUIET || echo "HTML_SOURCE environment variable not set..."
+  $QUIET || echo "OK, looking for HTML source file..."
+  if [ -f ../html/source ]; then
```

Comment:
> > +  if [ -f ../html/source ]; then
>
> How will this behave if running with CWD != script dir? (I guess I should test that...)

Good point. OK, fixed now so that the script should work as expected regardless of where it’s called from.